### DISCUSSION TITLE
[BUG] Theme is not applied to tabs

### DISCUSSION THREAD

**Discussion by luas10c** [ghostty-org/ghostty#4000] [Feature Requests, Ideas]
I believe it is a problem, after all the theme colors are not applied to the tabs.

This is the result I have
![image](https://github.com/user-attachments/assets/23a94715-521c-41fc-85ac-ba48566b411e)

**Comment by 00-kat** [comment]
```ini
window-theme = ghostty
```
Relevant docs: https://ghostty.org/docs/config/reference#window-theme.
Note that it doesn't work if you disable `gtk-titlebar` (which you have not done, but I'm just mentioning it).
(Note for anyone else who stumbles across this but has `gtk-titlebar` disabled: if you also have `window-decoration` disabled, re-enable `gtk-titlebar`, it'll work how you expected it to.)

**Reply by luas10c** [reply]
I changed the decorations style to ghostty's custom

**Reply by 00-kat** [reply]
What do you mean by "decorations style"?  Can you point me to which configuration entry you are talking about (maybe even consider sending your entire configuration)?

**Reply by luas10c** [reply]
https://ghostty.org/docs/config/reference#window-theme

ghostty - Use the background and foreground colors specified in the Ghostty configuration. This is only supported on Linux builds with Adwaita and gtk-adwaita enabled.

**Reply by 00-kat** [reply]
So this issue is still happening if you have `window-theme` set to `ghostty`?

**Reply by luas10c** [reply]
> Então esse problema ainda está acontecendo se você `window-theme`definiu como `ghostty`?

yes

**Reply by luas10c** [reply]
Another case where the theme is also not being applied

![image](https://github.com/user-attachments/assets/eaa92681-fce5-4e6a-993d-13fc33b7f78f)
